test/e2e/kubectl.go

```diff
@@ -238,6 +238,51 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl logs", func() {
+		It("should find a string in pod logs", func() {
+			mkpath := func(file string) string {
+				return filepath.Join(testContext.RepoRoot, "examples/guestbook-go", file)
+			}
+			controllerJson := mkpath("redis-master-controller.json")
+			nsFlag := fmt.Sprintf("--namespace=%v", ns)
+			By("creating Redis RC")
+			runKubectl("create", "-f", controllerJson, nsFlag)
```

Comment:
Add a call to `cleanup` in defer, so we aren't leaking resources. Same for `Kubectl patch` below.